### PR TITLE
Lesson4 4

### PR DESCRIPTION
**Pull request by kenldurham**
complete

### CODE REVIEW

File: css/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+html{
+   font-size:18px 
+
```
Comment (rtesagg84) [inline css/stylesheet.css:3]: Remove spacing!

File: css/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+html{
+   font-size:18px 
+   
+}
+
+body {
+
```
Comment (rtesagg84) [inline css/stylesheet.css:7]: Remove Spacing!

File: css/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    padding: 0;
+    color: black;
+    background: linear-gradient(white, blue);
+    text-align: center;
+    
+    }
+    .spacing{
+        margin: 10;
+        padding: 20;
+
```
Comment (rtesagg84) [inline css/stylesheet.css:18]: Remove spacing!

File: css/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    }
+.font{
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size: 1.5rem;
+}
+
+.font-2{
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size: 2.5rem;
+
```
Comment (rtesagg84) [inline css/stylesheet.css:28]: Remove spacing

File: css/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+}
+.font-3{
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size: 3.5rem;
+
```
Comment (rtesagg84) [inline css/stylesheet.css:33]: Remove Spacing!

File: css/stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  .bgcolor {
+   background-color:rgb(171, 161, 255);
+   border-block-style: groove;
+    writing-mode: horizontal-tb;
+    direction: unset;
+}
+  @media only screen and (max-width:370px) {
+    html{
+        font-size: 5px;
+
```
Comment (rtesagg84) [inline css/stylesheet.css:59]: Remove spacing!

**Review by rtesagg84 — COMMENTED** [review]
your code looks good, except the spacing i did get. And  one thing i  want to say  for you is make your own branch .all your work is on main branch witch is used by all the students  which is  difficult to  get your PR easily.

**Comment by kenldurham** [comment]
I have it on kennethdurham-main. Is that not there?

**Comment by kenldurham** [comment]
I was doing it wrong at first but it should be on kennethdurham-main

**Comment by kenldurham** [comment]
Here is the link
https://github.com/kenldurham/intro-to-programming-eclipse/tree/kennethdurham-main/instructions

**Pull request closed without merging**